parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -1377,6 +1377,7 @@ mod benches {
         [replace, Replace]
         [security, Security]
         [vault_registry, VaultRegistry]
+        [democracy, Democracy]
```

Comment:
Can you also start using the weight please? I.e. run the benchmark, create the weight file, and set the WeightInfo config item